docs/docs/index.md

```diff
@@ -0,0 +1,121 @@
+
+```sh
+make all
+```
+
+### Build images manually with Docker
+
+It's important to strictly follow the sequence. Several Docker images depend on each other.
+The full dependency graph for the images looks like:
+
+```mermaid
+flowchart LR
+    A(base) --> B(capella/base)
+    A(base) --> L(jupyter-notebook)
+    B(capella/base) --> C(t4c/client/base)
+    B(capella/base) --> D(capella/ease)
+    B(capella/base) --> M(capella/cli)
+    C(t4c/client/base) --> E(t4c/client/ease)
+    C(t4c/client/base) --> N(t4c/client/exporter)
+    C(t4c/client/base) --> O(t4c/client/backup)
+    B(capella/base) --> F(capella/remote) --> K(capella/remote/pure-variants)
+    C(t4c/client/base) --> G(t4c/client/remote) --> J(t4c/client/remote/pure-variants)
+    D(capella/ease) --> H(capella/ease/remote) --> I(capella/readonly)
+    style A fill:#ebb134,color:#000000
+    style B fill:#8feb34,color:#000000
+    style C fill:#34cceb,color:#000000
+    style D fill:#eb3477,color:#ffffff
+    style E fill:#eb3477,color:#ffffff
+    style F fill:#f2f1a7,color:#000000
+    style G fill:#f2f1a7,color:#000000
+    style H fill:#f2f1a7,color:#000000
+    style I fill:#d0a7f2,color:#000000
+    style J fill:#62f5f2,color:#000000
+    style K fill:#62f5f2,color:#000000
+    style L fill:#f5626c,color:#ffffff
+    style M fill:#f8fc03,color:#000000
+    style N fill:#2b2659,color:#ffffff
+    style O fill:#1703fc,color:#ffffff
+```
+
+Each highlighted color indicates the Dockerfile which is used to build the image:
+
+:material-checkbox-blank-circle:{ style="color: #ebb134 " } [Base](base.md) <br>
+:material-checkbox-blank-circle:{ style="color: #8feb34 " } [Capella Base](capella/base.md)<br>
+:material-checkbox-blank-circle:{ style="color: #f8fc03 " } [Capella CLI](capella/cli.md)<br>
+:material-checkbox-blank-circle:{ style="color: #34cceb " } [T4C Client Base](capella/t4c/base.md) <br>
+:material-checkbox-blank-circle:{ style="color: #eb3477 " } [EASE](capella/ease.md) <br>
+:material-checkbox-blank-circle:{ style="color: #f2f1a7 " } [Remote](remote.md) <br>
+:material-checkbox-blank-circle:{ style="color: #d0a7f2 " } [Capella read-only](capella/readonly.md) <br>
+:material-checkbox-blank-circle:{ style="color: #62f5f2 " } [pure::variants](capella/pure-variants.md) <br>
+:material-checkbox-blank-circle:{ style="color: #f5626c " } [Jupyter notebook](jupyter/index.md) <br>
+
+**Make sure that all `docker` commands are executed in the root directory of the repository.**
+
+For each image, you'll find documentation how to build & run the image manually.
+
+## Build images in a CI/CD environment
+
+We provide a [Gitlab CI/CD template](https://github.com/DSD-DBS/capella-dockerimages/blob/main/ci-templates/gitlab/image-builder.yml) to build and test the images in CI/CD environment.
+Please find the instructions [here](https://github.com/DSD-DBS/capella-dockerimages/tree/main/ci-templates/gitlab#image-builder).
```

Comment:
We probably also want to move the CI/CD instructions/docs to mkdocs or? So probably create an issue for that.

Reply:
Was not sure about it. But it would make sense, I guess.

Reply:
I've moved it: https://github.com/DSD-DBS/capella-dockerimages/pull/131/commits/3dad8928824426f87959eeef61d8a8e3bbd93016